src/components/proofs/EmailProof.tsx

```diff
@@ -133,11 +134,9 @@ export default function () {
             </div>
           </div>
           <button className={arrowContainer} onClick={() => setOpen(!open)}>
-            {!xs && (
-              <span className={getStartedText(open)}>
-                <span>{domain ? 'Set token' : 'Get started'}</span>
-              </span>
-            )}
+            <span className={getStartedText(open, displayFrom('xs'))}>
```

Comment:
Just put `displayFrom('xs')` inside the `getStartedText`, so you don't need to pass it as a function argument

Reply:
Changed